message_app/model.py

```diff
@@ -61,3 +78,100 @@ def get_last_user_id(cls) -> int:
         # Get last user ID
         id = User.query.order_by(User.id.desc()).first().id
         return id
+
+class Conversations(db.Model):
+    __tablename__ = 'conversations'
+    id = db.Column(db.Integer, primary_key=True)
+    participants = db.relationship("User", secondary=users_conversations, backref="conversations", lazy=True)
+    messages = db.relationship("Messages", backref="conversations")
+    created_at = db.Column(db.Float, default=time.time())
+    last_message_id = db.Column(db.Integer) # This column is a foreignkey
```

Comment:
Can you create a ticket and write down the ticket ID here?

Reply:
https://tan-tho-le.atlassian.net/jira/software/projects/VAJ/boards/1?selectedIssue=VAJ-32